jobs/focus_android_events_to_amplitude.sh

```diff
@@ -1,22 +1,20 @@
 #!/bin/bash
 
-if [[ -z "$date" || -z "$max_requests" || -z "$key_file" ]]; then
+if [[ -z "$date" || -z "$max_requests" || -z "$key_file" || -z "$artifact" ]]; then
     echo "Missing arguments!" 1>&2
     exit 1
 fi
 
-git clone https://github.com/mozilla/telemetry-streaming.git
-cd telemetry-streaming
-sbt assembly
+artifact_filename="artifact.jar"
+aws s3 cp "$artifact" "$artifact_filename"
```

Comment:
I'm annoyed that this is still an issue with EMR (client mode can't use s3 URLs directly).